app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    move, file_in, file_out = command.split(" ")
```

Comment:
You need to check if `mv` is the first token in the command as per task requirements